stock_lot_quantity/tests/test_lot_quantity.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+from openerp.addons.stock.tests.common import TestStockCommon
+from openerp.tools import mute_logger
+
+
+class TestLotQuantity(TestStockCommon):
+
+    @mute_logger('openerp.addons.base.ir.ir_model', 'openerp.models')
+    def test_00_picking_create_and_verify_lot_quantity(self):
+        """ Make inventories to add stock with the same production lot on differents locations. Verify stock of lot."""
+        LotObj = self.env['stock.production.lot']
+        stock_location_shelf_1 = self.ModelDataObj.xmlid_to_res_id('stock.stock_location_components')
+        stock_location_shelf_2 = self.ModelDataObj.xmlid_to_res_id('stock.stock_location_14')
+
+        # --------------------------------------------------------------
+        # Create first inventory to add lot with quantity on location 1.
+        # --------------------------------------------------------------
+        lot1_productB = LotObj.create({'name': 'Lot 1B', 'product_id': self.productB.id})
+        inventory = self.InvObj.create({'name': 'Test Lot Quantity',
+                                        'product_id': self.productB.id,
+                                        'filter': 'product'})
```

Comment:
Please use 'filter': 'none' and remove 'product_id' because you're inserting the inventory line yourself.